robotlib/include/robotlib/proximity_handler.hpp

```diff
@@ -4,13 +4,16 @@
 
 namespace robotlib::proximity_handler {
 
+    using namespace iointerface::sensor;
+
     class ProximityHandler {
     public: 
-        ProximityHandler(const double min_distance);
+        ProximityHandler(const UltraSonicSensor& us_sensor, const double min_distance);
```

Comment:
the `const` for a primitive type like double is useless.. because the value will be copied any way.